pyproject.toml

```diff
@@ -12,15 +12,19 @@ pyyaml = "^5.3.1"
 retry = "^0.9.2"
 paramiko = "^2.7.1"
 spurplus = "^2.3.3"
-cerberus = "^1.3.2"
+dataclasses-json = "^0.5.2"
+# portalocker for compatibility of Windows, trigger pywin32 installed
+portalocker = "^1.7.1" 
+azure-identity = {version = "^1.4.0", allow-prereleases = true}
```

Comment:
This was the one that made a bug in poetry appear, right?

Reply:
No, it's for compatible inside azure packages. check [here](https://docs.microsoft.com/en-us/azure/developer/python/azure-sdk-authenticate?tabs=cmd#using-defaultazurecredential-with-sdk-management-libraries)